marimo/_plugins/ui/_impl/input.py

```diff
@@ -54,6 +54,7 @@ class number(UIElement[Optional[Numeric], Optional[Numeric]]):
     - `stop`: the maximum value of the interval
     - `step`: the number increment
     - `value`: default value
+    - `debounce`: whether to debounce the number picker
```

Comment:
"whether to debounce (rate-limit) value updates from the frontend"?

Or something like that.